Controllers/api.controller.js

```diff
@@ -10,7 +10,17 @@ exports.getTopics = (req, res, next) => {
     .then((topics) => {
         res.status(200).send({ topics });
     })
+    .catch(next);  
```

Comment:
You've added this catch block to the previous task. Is it catching anything? If it is, where are the tests to accompany it?